### PR TITLE
fixes #86: eRegs N&C header updates

### LINKED ISSUE #86: Implement notice & comment header design
screen shot 2016-03-31 at 12 21 11 pm

^
Header should look roughly like this

### PR DESCRIPTION
**Pull request by donjo**
screen shot 2016-03-31 at 12 22 26 pm

v1 of the header design. Still need the reg title component

### CODE REVIEW

File: notice_and_comment/templates/regulations/main-header.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="main-head">
```
Comment (cmc333333) [inline notice_and_comment/templates/regulations/main-header.html]: Mind adding a `{% comment %}` to explain why this override is needed? I think for the "Partnership between" text?

**Comment by cmc333333** [comment]
Also: don't feel tied to the naming of files in `-site` if you're adding new files. If the divisions there make sense, you might want to mirror them in "custom" land, but there's no need to if another division makes more sense. The key is that their file names can't collide (hence we have a "custom" dir in ATF eregs)

File: notice_and_comment/static/regulations/css/less/module/header-custom.less [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    color: darken(@blue, 10%);;
+  }
+}
+
+/*
+Site Title
+================
+*/
+
+.title a:link,
```
Comment (xtine) [inline notice_and_comment/static/regulations/css/less/module/header-custom.less]: Can you combine the attributes like this:

`.title a {
  &:active,
  &:hover,
  &:link,
  &:visited {
  }
}`

File: notice_and_comment/static/regulations/css/less/layout-custom.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/*
+ Layout
+ ==========================================================================
+ layout.less defines the structural layout of the site
+ */
+
+.panel {
+    position: fixed;
+    top: 90px;
+    bottom: 0px;
```
Comment (xtine) [inline notice_and_comment/static/regulations/css/less/layout-custom.less]: No need to use a unit with 0.
https://pages.18f.gov/frontend/css-coding-styleguide/units/

**Comment by donjo** [comment]
@xtine, @cmc333333 thanks! updated. Also, I ended up stripping out some of that CSS to continue with the theme of only putting a style in the -custom.less files if it is modifying something from -site. Some of what I had there was just pasted over from the -site styles and I forgot to strip out the ones I wasn't changing.

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**